src/wp-admin/includes/dashboard.php

```diff
@@ -1544,7 +1544,7 @@ function wp_welcome_panel() {
 			/* translators: 1: link to ClassicPress FAQs page, 2: link to ClassicPress support forum */
 			__( 'If you need help with something else, please see our <a href="%1$s"><strong>FAQs page</strong></a>. If your question is not answered there, you can make a new post on our <a href="%2$s"><strong>support forum</strong></a>.' ),
 			'https://docs.classicpress.net/faq-support/',
-			'https://forums.classicpress.net/c/support/'
+			'https://forums.classicpress.net/c/supportc/support/'
```

Comment:
One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.

Reply:
:( bizarre, next time I do a manual S&R, automated clearly keeps messing things up.